bridge_adaptivity/bridge_lti/tests/test_provider.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from ddt import data, ddt
+import mock
+
+from bridge_lti.provider import learner_flow, lti_launch
+from module.models import Sequence
+from module.tests.test_views import BridgeTestCase
+
+log = logging.getLogger(__name__)
+
+
+@ddt
+class ProviderTest(BridgeTestCase):
+    def lti_launch_request_for_role(self, roles):
+        return type('', (object,), {
```

Comment:
Could we use `client.post` or `RequestFactory` instead of creating an unnamed class?